scripts/rageshake.go

```diff
@@ -110,6 +127,20 @@ func main() {
 	// Make sure bugs directory exists
 	_ = os.Mkdir("bugs", os.ModePerm)
 
+	// serve files under "bugs"
+	fs := http.FileServer(http.Dir("bugs"))
+	fs = http.StripPrefix("/api/listing/", fs)
+
+	// set auth if env vars exist
+	usr := os.Getenv("BUGS_USER")
+	pass := os.Getenv("BUGS_PASS")
+	if usr == "" || pass == "" {
+		fmt.Println("BUGS_USER and BUGS_PASS env vars not found. No authentication is running for /api/listing")
+	} else {
+		fs = basicAuth(fs, usr, pass, "Enter username and password")
```

Comment:
it's more than that. It is specifically used to identify things which should have the same u/p on a domain. Obviously if different apps with different auth all use "Enter username and password" that's no good.

Also, some clients may show "Enter the username and password for <realm>".